src/components/sections/LatticeView.tsx

```diff
@@ -64,6 +65,24 @@ const LatticeView = ({ compilationResult }: LatticeViewProps): JSX.Element => {
     const handleChange = () => {
         setCompilationText(!showCompilationText)
     }
+
+    // export slices to downloadable text JSON file
+    const exportJson = (slices: Slices) => {
+        const stringJson = JSON.stringify(slices)
+        const blob = new Blob([stringJson], { type: "text/plain;charset=utf-8" })
+        const url = window.URL || window.webkitURL
+        const link = url.createObjectURL(blob)
+        const a = document.createElement("a")
+        // Attach date suffix, plus random id for now. Can include circuit, params eventually
+        const rand_n = (Math.random() + 1).toString(36).substring(8)
+        const date = new Date().toISOString().split("T")[0]
+        a.download = "LSC_" + date + "_" + rand_n
```

Comment:
```suggestion
        a.download = "LSC_" + date + "_" + rand_n + ".json"
```
Relative to my [comment](https://github.com/latticesurgery-com/web-ui/pull/138#pullrequestreview-882825227) on #138.